packages/core/test/template-engine/built-in-extensions/query-builder/builder.spec.ts

```diff
@@ -167,3 +167,43 @@ it('Extension should throw error when no profile defined', async () => {
     `No profile name found`
   );
 });
+
+it('xtension should remove comments in sql statements', async () => {
```

Comment:
Typo: `xtension` should be `Extension`

Reply:
BTW, Please assist to add the multiple line comment test case, thanks !

Reply:
fixed